Applications/DisplayProperties/DisplayProperties.h

```diff
@@ -0,0 +1,50 @@
+#pragma once
+
+#include <AK/AKString.h>
+#include <AK/Vector.h>
+#include <AK/RefPtr.h>
+#include <LibCore/CConfigFile.h>
+#include <LibDraw/Color.h>
+#include <LibDraw/Size.h>
+#include <LibGUI/GWidget.h>
+
+struct ScreenSize
+{
+    int width;
+    int height;
+};
+
+class DisplayPropertiesWidget final
+{
+public:
+    enum class ButtonOperations
+    {
+        OK,
```

Comment:
Enums should be InterCaps, not ALLCAPS.
https://github.com/SerenityOS/serenity/blob/master/Documentation/CodingStyle.md